lib/factory_trace/tracker.rb

```diff
@@ -8,7 +8,7 @@ def initialize
 
     def track!
       ActiveSupport::Notifications.subscribe('factory_bot.run_factory') do |_name, _start, _finish, _id, payload|
-        name = payload[:name].to_s
+        name = payload[:factory].name.to_s
```

Comment:
Revert this too, because now, AFAIU, you store not the exact name factory was called, but the name of the factory.

In other words, when alias name is used `payload[:factory].name.to_s != payload[:name].to_s`

Reply:
Yes, and that introduces problem: I, for example I'll use factory only by alias, gem will warns that I'm not using factory (because it will not find it by name)

Reply:
No, sorry. You got me maybe wrong. I perfectly understand that this is a problem. I just want to not mix purposes of some classes.